sim_learn.py

```diff
@@ -103,76 +102,48 @@ def main(args):
     stderr.write('  model random seed:    0x%x\n' % seed_model)
     stderr.write('\n')
 
-    accuracy = array([])
-    training_times = array([])
-    iterations = array([])
-
+    # create different experiment instances
+    experiments = []
     for j in range(instances):
+        l_name = log_name + str(j)
+        experiment = ExperimentLogisticRegression(
+            log_name=l_name,
+            n=n,
+            k=k,
+            N=N,
+            seed_instance=seed_instance + j,
+            seed_model=seed_model + j,
+            transformation=transformation,
+            combiner=combiner
+        )
+        experiments.append(experiment)
+
+    experimenter = Experimenter(log_name, experiments)
+    # run the instances
+    experimenter.run()
+
+    # output format
+    str_format = '{:<15} {:<10} {:<8} {:<8} {:<8} {:<8} {:<12} {:<15} {:<6} {:<8} {:<8} {:<8}'
```

Comment:
fancy! :+1: should we use `\t` here instead of space? We do so on other occasions, but it may use more space if printed to the screen.

Reply:
Done that:
https://github.com/nils-wisiol/pypuf/blob/e1d408389cff95a58dbaeb7cc39e702b99b6d4be/sim_learn.py#L126